controllers/translateController.js

```diff
@@ -1,34 +1,62 @@
-const fs = require('fs');
-const { parsePath } = require('../utils/getConfigPath');
-const { validateSettingsFile } = require('../utils/validateSettingsFile');
-const { dset: setDeepValue } = require('dset');
-const { validateAndPromptUserJSONFiles } = require('../utils/validateAndPromptUserJSONFiles');
-const { setTranslateWithFallbackEngines, isEngineValid } = require('../services/translateService');
-const { validateLanguageRequested } = require('../utils/supportedLanguagesUtils');
-
-const translateController = async (text, sourceLanguage, nameOfTranslation, options) => {
+const fs = require("fs");
+const { parsePath } = require("../utils/getConfigPath");
+const { validateSettingsFile } = require("../utils/validateSettingsFile");
+const { dset: setDeepValue } = require("dset");
+const {
+  validateAndPromptUserJSONFiles,
+} = require("../utils/validateAndPromptUserJSONFiles");
+const {
+  setTranslateWithFallbackEngines,
+  isEngineValid,
+} = require("../services/translateService");
+const {
+  validateLanguageRequested,
+} = require("../utils/supportedLanguagesUtils");
+const { validEngines } = require("../utils/translationEnginesUtils");
+
+const translateController = async (
+  text,
+  sourceLanguage,
+  nameOfTranslation,
+  options,
+) => {
   const settingsFilePath = parsePath(options.settingsFile);
   validateSettingsFile(settingsFilePath);
   validateLanguageRequested(sourceLanguage, options.engine);
-  
-  if (!text) throw new Error('No text to translate provided');
-  if (!nameOfTranslation) throw new Error('No name of translation provided');
-  
-  const { languages, basePath, translationEngines: settingsTranslationEngines } = require(settingsFilePath);
 
-  if (options.engine && !isEngineValid(options.engine)) throw new Error(`You've provided an invalid engine as arg on your CLI Command. Try with one of these: ${validEngines.join(', ')}`);
+  if (!text) throw new Error("No text to translate provided");
```

Comment:
Not sure if text should be validated different, for example the length or the type

Reply:
changed